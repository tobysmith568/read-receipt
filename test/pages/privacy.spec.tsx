import { render } from "@testing-library/react";
import { Provider } from "jotai";
import PrivacyPolicyPage from "src/pages/privacy";

describe("Privacy", () => {
  it("should render the privacy policy", () => {
    const { container } = render(
      <Provider>
        <PrivacyPolicyPage />
      </Provider>
    );

    expect(container).toMatchSnapshot();
  });
});
