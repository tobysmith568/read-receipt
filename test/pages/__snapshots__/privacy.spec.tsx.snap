// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Privacy should render the privacy policy 1`] = `
<div>
  <div
    class="css-1wcee00"
  >
    <a
      class="css-7kai5d"
      href="/"
    >
      &lt;- Home
    </a>
    <div
      class="css-d1odxa"
    >
      <h1>
        Privacy Policy
      </h1>
      <p>
        Last updated: March 11, 2022
      </p>
      <p>
        This Privacy Policy describes Our policies and procedures on the collection, use and disclosure of Your information when You use the Service and tells You about Your privacy rights and how the law protects You.
      </p>
      <p>
        We use Your Personal data to provide and improve the Service. By using the Service, You agree to the collection and use of information in accordance with this Privacy Policy. This Privacy Policy has been created with the help of the

        <a
          href="https://www.freeprivacypolicy.com/blog/sample-privacy-policy-template/"
          rel="external nofollow noreferrer noopener"
          target="_blank"
        >
          Privacy Policy Template
        </a>
        .
      </p>
      <h1>
        Interpretation and Definitions
      </h1>
      <h2>
        Interpretation
      </h2>
      <p>
        The words of which the initial letter is capitalized have meanings defined under the following conditions. The following definitions shall have the same meaning regardless of whether they appear in singular or in plural.
      </p>
      <h2>
        Definitions
      </h2>
      <p>
        For the purposes of this Privacy Policy:
      </p>
      <ul
        class="css-kq9es3"
      >
        <li>
          <p>
            <strong>
              Account
            </strong>
             means a unique account created for You to access our Service or parts of our Service.
          </p>
        </li>
        <li>
          <p>
            <strong>
              Company
            </strong>
             (referred to as either "the Company", "We", "Us" or "Our" in this Agreement) refers to Read Receipt.
          </p>
        </li>
        <li>
          <p>
            <strong>
              Cookies
            </strong>
             are small files that are placed on Your computer, mobile device or any other device by a website, containing the details of Your browsing history on that website among its many uses.
          </p>
        </li>
        <li>
          <p>
            <strong>
              Country
            </strong>
             refers to: United Kingdom
          </p>
        </li>
        <li>
          <p>
            <strong>
              Device
            </strong>
             means any device that can access the Service such as a computer, a cellphone or a digital tablet.
          </p>
        </li>
        <li>
          <p>
            <strong>
              Personal Data
            </strong>
             is any information that relates to an identified or identifiable individual.
          </p>
        </li>
        <li>
          <p>
            <strong>
              Service
            </strong>
             refers to the Website.
          </p>
        </li>
        <li>
          <p>
            <strong>
              Service Provider
            </strong>
             means any natural or legal person who processes the data on behalf of the Company. It refers to third-party companies or individuals employed by the Company to facilitate the Service, to provide the Service on behalf of the Company, to perform services related to the Service or to assist the Company in analysing how the Service is used.
          </p>
        </li>
        <li>
          <p>
            <strong>
              Usage Data
            </strong>
             refers to data collected automatically, either generated by the use of the Service or from the Service infrastructure itself (for example, the duration of a page visit).
          </p>
        </li>
        <li>
          <p>
            <strong>
              Website
            </strong>
             refers to Read Receipt, accessible from

            <a
              href="https://read-receipt.tobythe.dev/"
              rel="external nofollow noreferrer noopener"
              target="_blank"
            >
              https://read-receipt.tobythe.dev/
            </a>
          </p>
        </li>
        <li>
          <p>
            <strong>
              You
            </strong>
             means the individual accessing or using the Service, or the company, or other legal entity on behalf of which such individual is accessing or using the Service, as applicable.
          </p>
        </li>
      </ul>
      <h1>
        Collecting and Using Your Personal Data
      </h1>
      <h2>
        Types of Data Collected
      </h2>
      <h3>
        Personal Data
      </h3>
      <p>
        While using Our Service, We may ask You to provide Us with certain personally identifiable information that can be used to contact or identify You. Personally identifiable information may include, but is not limited to:
      </p>
      <ul
        class="css-kq9es3"
      >
        <li>
          <p>
            Email address
          </p>
        </li>
        <li>
          <p>
            Usage Data
          </p>
        </li>
      </ul>
      <h3>
        Usage Data
      </h3>
      <p>
        Usage Data is collected automatically when using the Service.
      </p>
      <p>
        Usage Data may include information such as Your Device's Internet Protocol address (e.g. IP address), browser type, browser version, the pages of our Service that You visit, the time and date of Your visit, the time spent on those pages, unique device identifiers and other diagnostic data.
      </p>
      <p>
        When You access the Service by or through a mobile device, We may collect certain information automatically, including, but not limited to, the type of mobile device You use, Your mobile device unique ID, the IP address of Your mobile device, Your mobile operating system, the type of mobile Internet browser You use, unique device identifiers and other diagnostic data.
      </p>
      <p>
        We may also collect information that Your browser sends whenever You visit our Service or when You access the Service by or through a mobile device.
      </p>
      <h3>
        Tracking Technologies and Cookies
      </h3>
      <p>
        We do not use Cookies or similar tracking technologies to track the activity on Our Service.
      </p>
      <h2>
        Use of Your Personal Data
      </h2>
      <p>
        The Company may use Personal Data for the following purposes:
      </p>
      <ul
        class="css-kq9es3"
      >
        <li>
          <p>
            <strong>
              To provide and maintain our Service
            </strong>
            , including to monitor the usage of our Service.
          </p>
        </li>
        <li>
          <p>
            <strong>
              For the performance of a contract:
            </strong>
             the development, compliance and undertaking of the purchase contract for the products, items or services You have purchased or of any other contract with Us through the Service.
          </p>
        </li>
        <li>
          <p>
            <strong>
              To manage Your requests:
            </strong>
             To attend and manage Your requests to Us.
          </p>
        </li>
      </ul>
      <p>
        We may share Your personal information in the following situations:
      </p>
      <ul
        class="css-kq9es3"
      >
        <li>
          <strong>
            With Your consent
          </strong>
          : We may disclose Your personal information for any other purpose with Your consent.
        </li>
      </ul>
      <h2>
        Retention of Your Personal Data
      </h2>
      <p>
        The Company will retain Your Personal Data only for as long as is necessary for the purposes set out in this Privacy Policy. We will retain and use Your Personal Data to the extent necessary to comply with our legal obligations (for example, if we are required to retain your data to comply with applicable laws), resolve disputes, and enforce our legal agreements and policies.
      </p>
      <p>
        The Company will also retain Usage Data for internal analysis purposes. Usage Data is generally retained for a shorter period of time, except when this data is used to strengthen the security or to improve the functionality of Our Service, or We are legally obligated to retain this data for longer time periods.
      </p>
      <h2>
        Disclosure of Your Personal Data
      </h2>
      <h3>
        Law enforcement
      </h3>
      <p>
        Under certain circumstances, the Company may be required to disclose Your Personal Data if required to do so by law or in response to valid requests by public authorities (e.g. a court or a government agency).
      </p>
      <h3>
        Other legal requirements
      </h3>
      <p>
        The Company may disclose Your Personal Data in the good faith belief that such action is necessary to:
      </p>
      <ul
        class="css-kq9es3"
      >
        <li>
          Comply with a legal obligation
        </li>
        <li>
          Protect and defend the rights or property of the Company
        </li>
        <li>
          Prevent or investigate possible wrongdoing in connection with the Service
        </li>
        <li>
          Protect the personal safety of Users of the Service or the public
        </li>
        <li>
          Protect against legal liability
        </li>
      </ul>
      <h2>
        Security of Your Personal Data
      </h2>
      <p>
        The security of Your Personal Data is important to Us, but remember that no method of transmission over the Internet, or method of electronic storage is 100% secure. While We strive to use commercially acceptable means to protect Your Personal Data, We cannot guarantee its absolute security.
      </p>
      <h1>
        Children's Privacy
      </h1>
      <p>
        Our Service does not address anyone under the age of 13. We do not knowingly collect personally identifiable information from anyone under the age of 13. If You are a parent or guardian and You are aware that Your child has provided Us with Personal Data, please contact Us. If We become aware that We have collected Personal Data from anyone under the age of 13 without verification of parental consent, We take steps to remove that information from Our servers.
      </p>
      <p>
        If We need to rely on consent as a legal basis for processing Your information and Your country requires consent from a parent, We may require Your parent's consent before We collect and use that information.
      </p>
      <h1>
        Links to Other Websites
      </h1>
      <p>
        Our Service may contain links to other websites that are not operated by Us. If You click on a third party link, You will be directed to that third party's site. We strongly advise You to review the Privacy Policy of every site You visit.
      </p>
      <p>
        We have no control over and assume no responsibility for the content, privacy policies or practices of any third party sites or services.
      </p>
      <h1>
        Changes to this Privacy Policy
      </h1>
      <p>
        We may update Our Privacy Policy from time to time. We will notify You of any changes by posting the new Privacy Policy on this page.
      </p>
      <p>
        We will let You know via email and/or a prominent notice on Our Service, prior to the change becoming effective and update the "Last updated" date at the top of this Privacy Policy.
      </p>
      <p>
        You are advised to review this Privacy Policy periodically for any changes. Changes to this Privacy Policy are effective when they are posted on this page.
      </p>
      <h1>
        Contact Us
      </h1>
      <p>
        If you have any questions about this Privacy Policy, You can contact us:
      </p>
      <ul
        class="css-kq9es3"
      >
        <li>
          By email:
          <a
            href="mailto:contact@tobythe.dev"
          >
            contact@tobythe.dev
          </a>
        </li>
      </ul>
    </div>
    <div
      class="css-1g71zvr"
    >
      Copyright © Toby Smith
      the year
      .
      <a
        class="css-gy0gih"
        href="/terms"
      >
        Terms.
      </a>
      <a
        class="css-gy0gih"
        href="/privacy"
      >
        Privacy.
      </a>
      <a
        class="css-gy0gih"
        href="/licenses"
      >
        Third-party licenses.
      </a>
    </div>
  </div>
</div>
`;
